src/internal.c

```diff
@@ -2706,6 +2709,7 @@ static int DoNewKeys(WOLFSSH* ssh, byte* buf, word32 len, word32* idx)
                 break;
 
             case ID_AES128_CBC:
+            case ID_AES128_CTR:
```

Comment:
When using AesSetKey for an AES-CTR mode key, you need to use AES_ENCRYPTION.

Reply:
Fixed